### PR TITLE
Add setter + various fixes

### PR DESCRIPTION
**Pull request by william-kalvarisky**

### CODE REVIEW

File: src/Types/Filters.php [hunk 1]
```diff
@@ -6,17 +6,21 @@
  * Class Filters
  *
  * @method string getName()
- * @method FiltersValues[] getValue()
+ * @method FiltersValues[] getValues()
+ * @method Filters setName(string $name)
+ * @method Filters setValue(FiltersValues[] $values)
```
Comment (pfeyssaguet) [inline src/Types/Filters.php:11]: setValues()

File: src/Types/PaginatedProducts.php [hunk 2]
```diff
@@ -9,6 +9,10 @@
  * @method Pagination getPagination()
  * @method Product[] getProducts()
  * @method Product[] getTopOffers()
+ * @method PaginatedProducts setFilters(Filters[] $filter)
```
Comment (pfeyssaguet) [inline src/Types/PaginatedProducts.php:12]: $filters

File: src/Types/ActionFlags.php [hunk 3]
```diff
@@ -8,6 +8,9 @@
  * @method bool getIsAutoRenew()
  * @method bool getIsManualProvisioning()
  * @method bool getRenewalSku()
+ * @method ActionFlags setIsAutoRenew(bool $isAutoRenew)
+ * @method ActionFlags setIsManualProvisioning(bool $isManualProcessing)
+ * @method ActionFlags setRenewalSku(bool $isRenew)
```
Comment (pfeyssaguet) [inline src/Types/ActionFlags.php:13]: $renewalSku

File: src/Types/ArrowsphereIdentifier.php [hunk 4]
```diff
@@ -9,6 +9,10 @@
  * @method string getSkuXac()
  * @method string getSkuXsp()
  * @method string getOrderableSku()
+ * @method ArrowsphereIdentifier setSku(string $sku)
+ * @method ArrowsphereIdentifier setSkuXac(string $skuXac)
+ * @method ArrowsphereIdentifier setSkuXsp(string $skuXsp)
+ * @method ArrowsphereIdentifier setOrderableSku(string $OrderableSku)
```
Comment (pfeyssaguet) [inline src/Types/ArrowsphereIdentifier.php:15]: $orderableSku

File: src/Types/Program.php [hunk 5]
```diff
@@ -7,7 +7,10 @@
  *
  * @method bool getIsEnabled()
  * @method string getLegacyCode()
- * @method ProgramName getName()
+ * @method ProgramName getNames()
+ * @method Program setIsEnabled(bool $isEnabled)
+ * @method Program setLegacyCode(string $legacyCode)
+ * @method Program setNames(ProgramName $name)
```
Comment (pfeyssaguet) [inline src/Types/Program.php:13]: $names

File: src/Types/Uom.php [hunk 6]
```diff
@@ -7,6 +7,8 @@
  *
  * @method int getQuantity()
  * @method string getType()
+ * @method Uom setQuantity(int $qty)
```
Comment (pfeyssaguet) [inline src/Types/Uom.php:10]: $quantity

**Review by pfeyssaguet — CHANGES_REQUESTED** [review]

File: src/Types/Filters.php (hunk 1 above)
Comment (william-kalvarisky) [reply src/Types/Filters.php]: done

File: src/Types/PaginatedProducts.php (hunk 2 above)
Comment (william-kalvarisky) [reply src/Types/PaginatedProducts.php]: done

File: src/Types/ActionFlags.php (hunk 3 above)
Comment (william-kalvarisky) [reply src/Types/ActionFlags.php]: done

File: src/Types/Program.php (hunk 5 above)
Comment (william-kalvarisky) [reply src/Types/Program.php]: done

File: src/Types/Uom.php (hunk 6 above)
Comment (william-kalvarisky) [reply src/Types/Uom.php]: done

File: src/Types/Filters.php [hunk 7]
```diff
@@ -6,17 +6,21 @@
...
+ * @method Filters setName(string $name)
+ * @method Filters setValues(FiltersValues[] $values)
  */
 class Filters extends AbstractType
 {
     public const NAME = 'name';
 
+    public const VALUES = 'values';
+
     public const VALUE = 'value';
```
Comment (pfeyssaguet) [inline src/Types/Filters.php:19]: This const can be removed now, no?

**Review by pfeyssaguet — APPROVED** [review]

**Pull request merged**